src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PollingWildCardChangeToken.cs

```diff
@@ -178,6 +178,10 @@ private void ComputeHash(IncrementalHash sha256, string path, DateTime lastChang
             sha256.AppendData(_byteBuffer, 0, length);
             sha256.AppendData(Separator, 0, Separator.Length);
 
+#if NET
```

Comment:
I think you could change this for all builds to be:
```C#
BinaryPrimitives.WriteInt64LittleEndian(_byteBuffer, lastChangedUtc.Ticks);
```
The endianness shouldn't matter here, as it's just being used in an in-memory hash to determine whether something changes or not.

Could probably also use:
```C#
long ticks = lastChangedUtc.Ticks;
MemoryMarshal.Write(_byteBuffer, ref ticks);
```

Reply:
> Could probably also use:

I was avoiding this one because it is a candidate for "requires unsafe" due to its generic nature